### DISCUSSION TITLE
Regex Support for Org and Repo Filterting

### DISCUSSION THREAD

**Discussion by arunparaman** [community/community#14581] [Other Feature Feedback, Questions, & Ideas]
Support regular expressions or partial matching for organization and Repository in GitHub Code Search would be greatly helpful to search across multi orgs and repos
org:^PATTERN
repo:^PATTERN

**Comment by look** [comment]
Thanks for the feedback, @arunparaman. We are working on some new backend features that should make this possible soon. Stay tuned!
